### ISSUE TITLE
Firefox: dotted outline around focused inputs

### ISSUE THREAD

**Issue by silverwind** [picturepan2/spectre#51]
These may be not intended:

![](https://cloud.githubusercontent.com/assets/115237/15492649/f1b3e35e-217d-11e6-807e-a513cf750422.png)

**Comment by Randore** [comment]
Adding  { outline: medium none; } will work.

**Comment by jkochis** [comment]
Please consider leaving the outline as it is. This is an important feature for users without a pointing device, such as a mouse or tap interface. If you want to remove it, at least provide alternative styling.

For more information:
http://www.outlinenone.com/

**Comment by silverwind** [comment]
The border-color is already changed to indicate focus, so I'd say it's fine to remove the outline:

https://github.com/picturepan2/spectre/blob/d1c1567f04ad990b361f1780c6ad71cb3ab2cfe7/src/forms.less#L24

**Comment by nder** [comment]
Also [`outline` is already cleared](https://github.com/picturepan2/spectre/blob/d1c1567f04ad990b361f1780c6ad71cb3ab2cfe7/src/forms.less#L18).

I guess it's specific to Firefox `::-moz-focus-inner` behaviour, which should be easy to overcome (http://stackoverflow.com/questions/71074/how-to-remove-firefoxs-dotted-outline-on-buttons-as-well-as-links).

**Comment by silverwind** [comment]
I've tried various `-moz` rules, but none seem to work in this case. What does work is adding `!important` to `outline: 0`. Really not sure why, the outline of `input` is not an issue with other stylesheets.

**Comment by silverwind** [comment]
Found the cause of this outline in normalize.css:

```css
/**
 * Remove the inner border and padding in Firefox.
 */
button::-moz-focus-inner,
input::-moz-focus-inner {
  border: 0;
  padding: 0;
}
/**
 * Restore the focus styles unset by the previous rule.
 */
button:-moz-focusring,
input:-moz-focusring {
  outline: 1px dotted ButtonText;
}
```

The issue was fixed in https://github.com/necolas/normalize.css/commit/b1f3b0c0aa3da28019bf8091fafd1699bff9aec3, which is released in v4.1.1.

**Comment by picturepan2** [comment]
I fixed it in https://github.com/picturepan2/spectre/pull/73/commits/ffe5499053f4d9275808cfe0a7e4903bfedb9023. Thanks.
